docs/explore/airnode/potential-obstacles.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Potential obstacles of operating oracle nodes
+sidebarHeader: Explore
+sidebarSubHeader:
+pageHeader: Explore → Airnode
+path: /explore/airnode/potential-obstacles-of-operating-nodes.html
+outline: deep
+tags:
+---
+
+<PageHeader/>
+
+<SearchHighlight/>
+
+# {{$frontmatter.title}}
+
+As a part of its previous efforts, API3 communicated with API providers extensively. 
+We observed a number of obstacles that might prevent them from oracle onboarding 
+and operation, the most important of which we have outlined below.
+
+## Lack of familiarity
+
+Traditional API providers are typically not more familiar with blockchain
+technologies than the general public. This applies even for the ones that
+curate cryptocurrency market data—as their main operation is collecting data
+from exchange APIs, processing them, and serving the result through their own 
+APIs—which does not require any blockchain-specific know-how. Therefore, they 
```

Comment:
`APIs which` instead of `APIs—which`?

Reply:
I will look at this after merging this PR